src/components/ReviewFullPage/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import ReviewCard from "../ReviewCard";
+import { getReviewById } from "../../utils/apis";
+import "./styles.css";
+
+function ReviewFullPage() {
+  const [review, setReview] = useState(null);
+  const [loading, setLoading] = useState(true);
+
+  const reviewId = useParams().review_id;
+
+  useEffect(() => {
+    setLoading(true);
+    getReviewById(reviewId)
+      .then((review) => {
+        setReview(review);
+        setLoading(false);
+      })
+      .catch((error) => {
+        console.log(error);
+        setLoading(false);
+      });
+  }, [reviewId]);
+
+  if (loading || review === null) {
+    return <div>Loading...</div>;
+  }
+  return (
+    <div>
```

Comment:
To save adding another `<div>` to the mix, while still being able to wrap the JSX you're returning in one element, I'd suggest using a React fragment, i.e <> </> instead, which will disappear when you actually render the page ✨